src/utils/removeListening.ts

```diff
@@ -0,0 +1,8 @@
+import { NICKNAME_SUFFIX } from "../constants/messagingConstants";
+
+export const removeListening = (nick: string) => {
+  const emojiRemovedNick = nick.replace("🎧", "");
+  const separator = NICKNAME_SUFFIX;
+  const newNick = emojiRemovedNick.split(new RegExp(separator, "i"));
+  return newNick[0].trim();
+};
```

Comment:
Can it be **suffix** instead of **nick** at every place it's being used ? 
-> **emojiRemovedSuffix**
-> **newSuffix**

Reply:
its nickname only 
we are removing emoji from nick I dont think it needs to change the name